Detectors/TPC/calibration/src/CorrectionMapsLoader.cxx

```diff
@@ -296,4 +298,27 @@ void CorrectionMapsLoader::updateInverse()
   }
 }
 
+float CorrectionMapsLoader::getMapMeanRate(const o2::gpu::TPCFastTransform* mp, bool lumiOverridden) const
+{
+  float val = 0;
+  if (getLumiScaleType() == 1) { // CTP Lumi
+    val = mp->getLumi();
+  } else if (getLumiScaleType() == 2) { // TPC Scaler (IDC)
+    val = mp->getIDC();
+    if (val < 0. && mp->getLumi() >= 0. && !lumiOverridden) {
+      if (mp->getLumi() < o2::tpc::CorrMapParam::Instance().CTP2IDCFallBackThreshold) {
+        val = mp->getLumi();
+        static bool reportFallBack = true;
+        if (reportFallBack) {
+          reportFallBack = false;
+          LOGP(warn, "IDC scaling is requested but map IDC record is empty. Since map Lumi={} is less than fall-back threshold {}, interpret Lumi record as IDC",
+               val, o2::tpc::CorrMapParam::Instance().CTP2IDCFallBackThreshold);
+        }
```

Comment:
Wouldn't it be better that this goes to `getIDC()` (except the `lumiOverridden`). Then one can directly call the `getIDC()` also in other places and get the fallback automatically.
Also, the ref map in case of derivative can have `IDC < 0`, perhaps it would be better to set the default to `0`, not `-1` and check that?

Reply:
@wiechula this means that the getter should have a signature (assuming we don't want to introduce a dependency of the TPCFastTransform on TPCCalibration lib)
```
TPCFastTransform::getIDC(bool lumiOverridden, float fallbackThreshold)
```
If you think it is better, I can change this.
Concerning the defaults: I did not want to give a special meaning to 0, but ok, will change both defaults to 0.